mesonbuild/modules/gnome.py

```diff
@@ -1518,7 +1518,7 @@ def mkenums_simple(self, state, args, kwargs):
             fhead += '%s\n' % body_prefix
         fhead += '#include "%s"\n' % hdr_filename
         for hdr in sources:
-            fhead += '#include "%s"\n' % os.path.basename(str(hdr))
+            fhead += '#include "{}"\n'.format(os.path.basename(str(hdr)))
```

Comment:
This could be an f-string, right?

Reply:
Many things could be f-strings, but I exercise my highly opinionated judgment in deciding when stuff is "too complicated" to look good in an f-string. :D